cranelift/filetests/filetests/isa/aarch64/constants.clif

```diff
@@ -164,7 +164,7 @@ block0:
 }
 
 ; block0:
-;   orr x0, xzr, #4294967295
+;   movn x0, #0
```

Comment:
What caused this change?

Reply:
The returns are going through a different path now, so the immediate handling is a little different; but this is still correct I think (`movn x0, #0` generates all-ones, which for an `i32` func is the same as `u32::MAX` in the return register).